### ISSUE TITLE
Admin: add vat info to registrar data

### ISSUE THREAD

**Issue by vohmar** [internetee/registry#623]
For foreign registrars we need to specify whether the registrar is VAT liable.

/admin/registrars/??/edit
New fields to registrar profile are necessary
General block:
* vat %

vat nr field exists in registrar profile - optional

* if address country code is EE then vat% and vat no. values are optional. VAT is registry_vat_rate on the invoice. the value in the vat% field is ignored.
* if address country code is not EE and VAT no. value is missing, vat% value cannot be NULL (range 0-99). VAT is added to the invoice according to the specified vat% value.
* if address country code is not EE and VAT no. value is filled, vat% value is optional. VAT is not applied nor displayed on the invoice.
- Fill line item's "VATCode" with registrar VAT number if present
- Fill invoice's "TotalVAT"

**Comment by artur-beljajev** [comment]
@vohmar It seems VAT country is a required field. Can I use "registrars.country_code" (http://prntscr.com/h9h7hb) to populate existing registrars with VAT country?

**Comment by artur-beljajev** [comment]
See #509

**Comment by artur-beljajev** [comment]
@vohmar `registrars.vat` (a boolean) is not used at all. It has been imported from Fred. Do we need that data? If we do, explain how it will be used.

**Comment by vohmar** [comment]
registrars.vat can be removed

>> It seems VAT country is a required field. Can I use "registrars.country_code" (http://prntscr.com/h9h7hb) to populate existing registrars with VAT country?

lets drop the vat_coutry_code.

**Comment by artur-beljajev** [comment]
`registrars.vat` is removed.

**Comment by artur-beljajev** [comment]
#367 needs to be addressed along with this ticket

**Comment by vohmar** [comment]
if you can solve this in 2 hours the go for it, otherwise we will postpone #367

**Comment by artur-beljajev** [comment]
I messed up the tickets: the one I need is #407. #367 is also important, but can be postponed.

**Comment by vohmar** [comment]
do not see an option to set vat% under a registrar profile in admin (/admin/registrars/52/edit)

**Comment by artur-beljajev** [comment]
This ticket is done in the same branch as #509, which is not ready yet. What's why this ticket is still assigned to me.

**Comment by artur-beljajev** [comment]
e068442409e2908497bf5793ba7ee9403c318d88

**Comment by artur-beljajev** [comment]
Would you like to see registry's VAT rate as the default pre-filled of `VAT rate` form field?

**Comment by artur-beljajev** [comment]
https://github.com/internetee/registry/blob/ef55560d0ef0c050acb0c6d6c233a4900dd48ceb/app/models/bank_transaction.rb#L91

**Comment by artur-beljajev** [comment]
https://github.com/internetee/registry/blob/b68fe6e3124b28f12d805247d48d91212e5c41c3/app/models/directo.rb#L37

**Comment by vohmar** [comment]
registrar vat rate should have default value of regsitry vat rate as most of the registars are local

**Comment by artur-beljajev** [comment]
Test: 
- Balance top-up in registrar area
- Directo export
- Payment matching

**Comment by artur-beljajev** [comment]
Blocked by #770. I need to have it master before I can squash this one.

**Comment by vohmar** [comment]
#765 is in master

**Comment by artur-beljajev** [comment]
Has #772 merged.

**Comment by vohmar** [comment]
cannot set new invoice to be paied using admin "payment received" functionality
```
Apr 12 10:23:27 staging ADMIN[10679]: [staging-adm.infra] [8b7135d0-02b6-4ddf-a5ae-d7aec86a410f] [192.168.6.34]   Invoic
e Load (0.7ms)  SELECT  "invoices".* FROM "invoices" WHERE "invoices"."id" = $1 LIMIT 1  [["id", 71]]
Apr 12 10:23:27 staging ADMIN[10679]: [staging-adm.infra] [8b7135d0-02b6-4ddf-a5ae-d7aec86a410f] [192.168.6.34] Completed 500 Internal Server Error in 22ms (ActiveRecord: 1.4ms)
Apr 12 10:23:27 staging ADMIN[10679]: [staging-adm.infra] [8b7135d0-02b6-4ddf-a5ae-d7aec86a410f] [192.168.6.34] #012NoMethodError (undefined method `amount' for #<Invoice:0x0000000abbb368>):#012  app/controllers/admin/bank_statements_controller.rb:27:in `new'
```

minor issues - will make separate bug ticket for these to move this ticket forward: 
* If address country not EE and vat no exists admin returns error "VAT rate must be blank" in case there is a value in vat rate field. Expected - vat rate value is optional in this scenario - no errors should be returned here.
* if address country EE admin returns error "VAT rate must be blank" in case there is a value in vat rate field. Expected - vat rate value is optional in this scenario - no errors should be returned here.

**Comment by artur-beljajev** [comment]
1. Fixed
2, 3. We had a chat about this before. As far as I remember, there was no point of that data under such circumstances.

**Comment by vohmar** [comment]
no1: much worse now. Cannot create new registrar nor open details of any existing one. Internal error

```
Apr 12 16:09:07 staging ADMIN[1553]: [staging-adm.infra] [2123beff-eb09-41b4-985a-6c85c2bf037e] [192.168.6.34] Completed
 500 Internal Server Error in 21ms (ActiveRecord: 1.3ms)
Apr 12 16:09:07 staging ADMIN[1553]: [staging-adm.infra] [2123beff-eb09-41b4-985a-6c85c2bf037e] [192.168.6.34] #012Actio
nView::Template::Error (missing attribute: vat_rate):#012     8:             <dd><%= registrar.vat_no %></dd>#012     9:

 #012    10:             <dt><%= Registrar.human_attribute_name :vat_rate %></dt>#012    11:             <dd><%= number_
to_percentage(registrar.vat_rate, precision: 1) %></dd>#012    12: #012    13:             <dt><%= Registrar.human_attri
bute_name :accounting_customer_code %></dt>#012    14:             <dd><%= registrar.accounting_customer_code %></dd>#01
2  app/views/admin/registrars/_billing.html.erb:11:in _app_views_admin_registrars__billing_html_erb___42914055196591032
03_37670880'#012  app/views/admin/registrars/show.html.erb:35:in_app_views_admin_registrars_show_html_erb__15181373162
62299086_38411940'
```

no test coverage?
